Example/Example/SeriesDetailViewController.swift

```diff
@@ -9,6 +9,61 @@
 import Foundation
 import BothamUI
 
-class SeriesDetailViewController : ExampleViewController {
+private struct Config {
+    static let numberOfColumns = 3
+    static let cellHeight = 174
+    static let headerHeight = 534
+    static let footerHeight = 20
+    static let cellMargin = 20
+}
+class SeriesDetailViewController : ExampleViewController, BothamCollectionViewController, SeriesDetailUI {
```

Comment:
If we change the definition of BothamCollectionViewController to

we could maybe declare the datasource directly as SeriesDetailCollectionViewDataSource

```Swift
public protocol BothamCollectionViewController: class {
    var collectionView: UICollectionView! { get }
    typealias CollectionViewDataSourceType: BothamDataSource
    var dataSource: CollectionViewDataSourceType! { get set }
}


extension BothamCollectionViewController {
    public func showItems(items: [CollectionViewDataSourceType.ItemType]) {
        dataSource.items = items
        collectionView.reloadData()
    }
}
```